ansible/cleanup.yml

```
@@ -0,0 +1,74 @@
+---
+- hosts: production
+  become: yes
+  tasks:
+    - name: Stop any running education-website service
+      systemd:
+        name: education-website
+        state: stopped
+        enabled: no
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)